cluster/manifests/external-dns/deployment.yaml

```diff
@@ -16,20 +16,21 @@ spec:
     metadata:
       labels:
         application: external-dns
-        version: v0.5.3
+        version: v0.5.4
       annotations:
         iam.amazonaws.com/role: "{{ .LocalID }}-app-external-dns"
     spec:
       priorityClassName: system-cluster-critical
       containers:
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:v0.5.3
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.5.4
         args:
         - --source=service
         - --source=ingress
         - --provider=aws
         - --registry=txt
         - --txt-owner-id={{ .Region }}:{{ .LocalID }}
+        - --txt-cache-interval=1h
```

Comment:
I suggest we fix the problem mentioned here: https://github.com/kubernetes-incubator/external-dns/pull/612#issuecomment-399253828 before we enable caching in our setup.